### PR TITLE
Update SourceKitten to 0.15.0

### LINKED ISSUE #677: Should the latest version of SourceKitten be used?
It *seems* that jazzy is currently targeting SourceKitten from about 6 months ago. Would it make sense to upgrade/could it potentially fix issues with jazzy/SourceKit crashes (I haven't been able to run it on my proj for months, without crashing)?

### PR DESCRIPTION
**Pull request by wmcbain**
This should resolve #677

### CODE REVIEW

**Comment by segiddins** [comment]
@orta might be interested in what danger did to the comment here

**Comment by orta** [comment]
Ah yeah, needs updating to 4.0.1 - GitHub did an update to their markdown parser to be pure [commonmark](https://twitter.com/DangerSystems/status/800997175526203392)

**Comment by istx25** [comment]
Hi @wmcbain. Thanks for making this change. Please make Travis go green! and then we can merge this in. :)

**Comment by wmcbain** [comment]
Hey @istx25, did a bit more digging on this and realized I needed to rebuild integration fixtures. Doing that resolved a number of errors, and I'll need to push a commit to that repo as well as part of this PR.

However I'm hitting on errors from including a podspec attribute which I referenced here on issue #674. Should resolving that be included here? And would this also mean that I need to update Xcode and Swift versions in travis.yaml?

[Output](https://gist.github.com/wmcbain/32d2bd7a4500d0a07b5292895d4396d8) of running specs after rebuilding fixtures.

**Comment by jpsim** [comment]
I've updated the fixtures in edef43f. @wmcbain could you please rebase this?

File: CHANGELOG.md [hunk 1]
```diff
@@ -9,6 +9,20 @@
 * Podspec-based documentation respects the `swift-version` config option.  
   [Orta Therox](https://github.com/orta)
 
+## 0.7.23
```
Comment (wmcbain) [inline CHANGELOG.md]: Hit a snag with DangerCI requesting a CHANGELOG, should I not be including these changes?

**Comment by wmcbain** [comment]
Rebased against `realm/jazzy:master`.

**Comment by jpsim** [comment]
Thanks for this @wmcbain! There are some changes in SourceKitten `master` that would be useful to incorporate in jazzy too, so I'll cut a new release of SourceKitten (`0.15.1`), and then make a new PR towards jazzy incorporating that new release.

**Pull request closed without merging**